playbooks/FortiManager(FMG)-Triggers/On ZTP Profile Assignment in Device Record.json

```diff
@@ -0,0 +1,557 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "On ZTP Profile Assignment in Device Record",
```

Comment:
@fortijames Trigger condition is missing in trigger step 

<img width="565" alt="image" src="https://github.com/fortinet-fortisoar/solution-pack-fortimanager-ztp-framework/assets/46894140/a9059c3d-6ee4-4c87-ad35-b5869dbb503c">